minimal/compose/nginx/production/nginx.conf

```diff
@@ -36,7 +36,15 @@ http {
         listen 80;
         listen [::]:80;
         server_name {{ DOMAIN }};
-        return 301 https://$server_name$request_uri;
+
+        # Certbot always performs validations over HTTP
+        location ^~ /.well-known  {
```

Comment:
@pslobo `In fact, I couldn't force certbot to validate on https so that location is never even used` are you saying certbot can make it so that we don't need `location ^~ /.well-known` and  `location ^~ /.well-known` and we can just leave `return 301 https://$server_name$request_uri;` as it was?? if that is the case i would prefer that approach